Shippo/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net40"/>
+  <package id="Newtonsoft.Json" version="10.0.1" targetFramework="net40" />
```

Comment:
just putting a comment here to confirm that the new version of the json library doesn't break anything